regimes/at/identitites.go

```diff
@@ -0,0 +1,82 @@
+package at
+
+import (
+	"errors"
+	"regexp"
+	"strconv"
+
+	"github.com/invopop/gobl/cbc"
+	"github.com/invopop/gobl/i18n"
+	"github.com/invopop/gobl/org"
+	"github.com/invopop/validation"
+)
+
+const (
+	// IdentityKeyTaxNumber represents the Austrian tax number (Steuernummer) issued to
+	// people that can be included on invoices inside Austria. For international
+	// sales, the registered VAT number (Umsatzsteueridentifikationsnummer) should
+	// be used instead.
+	IdentityKeyTaxNumber cbc.Key = "at-tax-number"
+)
+
+// https://www.oecd.org/content/dam/oecd/en/topics/policy-issue-focus/aeoi/austria-tin.pdf
+
+var badCharsRegexPattern = regexp.MustCompile(`[^\d]`)
+
+var identityKeyDefinitions = []*cbc.KeyDefinition{
+	{
+		Key: IdentityKeyTaxNumber,
+		Name: i18n.String{
+			i18n.EN: "Tax Number",
+			i18n.DE: "Steuernummer",
+		},
+	},
+}
+
+func normalizeTaxNumber(id *org.Identity) {
```

Comment:
There needs to be a generic normalize method. Naming also needs to be consistent, so this should be `normalizeIdentity`.